utils/src/api_schema/api_schema.json

```diff
@@ -534,6 +544,8 @@
       "properties": {
         "name":       { "type": "string" },
         "dimensions": { "$ref": "#/definitions/refInt" },
+        "metric":     { "$ref": "#/definitions/metricFormatString" },
```

Comment:
Was this already implemented and only the JSON check was missing?

Reply:
yes